source/manual/get-started.html.md

```diff
@@ -161,11 +161,17 @@ Once your VM is running, you can SSH into it with:
 
 ### Set your Git username and email
 
-You can assign your name and email to commits on the VM:
+You can associate your git credentials with commits on the VM by setting your global configs on the dev machine.
 
     dev$ git config --global user.email "friendly.giraffe@digital.cabinet-office.gov.uk"
     dev$ git config --global user.name "Friendly Giraffe"
 
+It's worth noting at this point that if you ever need to destroy and rebuild your machine, you'll need to follow these steps again. Therefore, it's probably worth creating a file on your host machine (e.g. `~/.bash_profile_vm`) and writing these commands in there so that you can easily transfer them into your VM when needed.
+
+    mac$ more ~/.bash_profile_vm | pbcopy
+    dev$ vi ~/.bash_profile # then paste the contents in
+    dev$ source ~/.bash_profile # load it for the current session (or 'exit' and 'vagrant ssh' again)
```

Comment:
Not quite related, but have you considered trying to use Vagrant to manage some of this configuration? You can customise the configuration (e.g. [1]) like memory usage, and I can see being able to specify the Git user details be useful.

1: https://github.com/alphagov/govuk-puppet/blob/master/development-vm/Vagrantfile.localconfig.example

Reply:
Thanks Chris - I'll take a look.

Reply:
Raised https://github.com/alphagov/govuk-puppet/pull/9212